modules/ui/ModelTab.py

```diff
@@ -235,8 +235,8 @@ def __create_base_components(
 
         if has_vae:
             # base model
-            components.label(self.scroll_frame, row, 0, "VAE",
-                             tooltip="Directory or Hugging Face repository of a VAE model. Can be used to override the base model VAE.")
+            components.label(self.scroll_frame, row, 0, "VAE (Read Tooltip)",
+                             tooltip="Directory or Hugging Face repository of a VAE model in diffusion format. Can be used to override the base model VAE. Using a safetensor VAE file will cause an error that the model cannot be loaded.")
```

Comment:
The format is called diffusers, not diffusion.

Reply:
I had that initially, but google searching made me go to diffusion.  Will fix.